### ISSUE TITLE
Revise plan Download (formerly Export) page

### ISSUE THREAD

**Issue by stephaniesimms** [DMPRoadmap/roadmap#498]
UX wireframes @JEK-III : http://h90c3m.axshare.com/#g=1&p=write_plan
Functional Specification (global spec for all write plan pages): Implement with a non-table layout; attach labels to input fields
Related issues for write plan pages #491 #494 #495 #497

- Tab renamed "Download"
- Button renamed "Download Plan"
- Replace included elements options w/new, abbreviated menu of options; blue notepad w/functional specs - Fieldset legend for checkboxes
- by default export all answers (no tickbox for this)
- Project Details encompasses all info captured on Project Details page (still to-do - coversheet design for this info, related to #436)
- note functional spec for when/how to display unanswered questions tickbox
- add tooltip beside supplemental sections tickbox
- set default to most common file format (assuming PDF @sjDCC ? if not PDF most likely DOCX)

**Comment by briri** [comment]
@JEK-III there is no message displayed when I click on the blue notepad icon next to the supplemental sections checkbox. Are 'supplemental sections' customized sections of a funder template? If so should this checkbox be hidden when the plan does not have customized sections?

Use the following format for the tooltip button next to the supplemental sections checkbox:
```ruby
<a href="#" data-toggle="popover" rel: "popover" data-html: "true" role="button" 
             data-content="<%= _('Here's our helpful message') %>" 
             aria-label="<%= _('More information: Supplemental sections') %>">
          <span class="fa fa-question-circle" aria-hidden="true"></span></a>
```

**Comment by xsrust** [comment]
@JEK-III
As brian mentioned above, the notepad icon next to supplemental sections is blank.  Is the intended behavior to hide customized sections, and only display on customized templates?

Additionally, how are plans with multiple phases supposed to be handled? I dont know of any with more than 3 phases in the current dataset.  Perhaps this could be a drop-down box which appears when the plan has multiple phases?

**Comment by JEK-III** [comment]
@xsrust 
The intended behavior is to only show that checkbox on customized templates. I adjusted the way elements are layered in the wireframe so the annotation should work now.

As far as multi-phase templates go, I see nothing wrong with the current approach of having separate tabs for each phase, labelled with the name of the phase. (In which case there will be no tab labeled "write plan")

**Comment by xsrust** [comment]
@JEK-III Thanks for that.  The wireframe looks good to me on the guidance/annotation front.

The question I intended to ask was about the download page with multiple pages.   The current behavior is to export only one phase at a time, since phases represent multiple stages in an application.  The wireframe on the Downloads page does not have any way to select which phase should be downloaded.

**Comment by dsisu** [comment]
I understand this. We applied our own styling to the proposed generic tool and the end-result isn’t great. I guess this is work in progress so I’m not too concerned about the DIY look of the current generic tool. The one thing that does concern me is the introduction of new elements on the page e.g. Explore. This is not a matter of styling and will have a big impact on the way people navigate the site. I’m not convinced this is more user-friendly than what we currently have and I don’t think it’s justified by accessibility either. We should not introduce changes like these without testing first. Also, accessibility and usability are not incompatible with good looks. It is important that we produce a well finished, professional looking generic tool that will entice organisations to download it and use it. I guess I’m preaching to the converted. I have other work to do now, I’ll look at the proposed wireframes next week and get back to you by Wednesday next week. Have a good weekend!

**Comment by stephaniesimms** [comment]
@xsrust raises a good point. @JEK-III - can you add a phase selector to the download options? this would only appear for templates w/multiple phases (eg H2020) and ideally include the specific phase names.

**Comment by JEK-III** [comment]
@xsrust Got it– I added a phase droplist to the download panel.

**Comment by xsrust** [comment]
Thanks @JEK-III

**Comment by vyruss** [comment]
Tagging issues #154 and #346 because relevant code will affect them in the future

**Comment by sjDCC** [comment]
The select phase to download works fine. You can only select one at a time, whereas I think we are exporting all 'complete' phases for the public DMPs page. I guess we don't need to let users export multiple at once though. What do you think @stephaniesimms?

A couple of suggestions:
- the 'questions as headings' concept is a difficult one to grasp. Could we rephrase this to "full question text as well as section headings"
- change 'supplimental section(s)...' to 'supplementary section(s)...'

There are a couple of places where we could do with extra spacing e.g. before 'format' or better alignment e.g. the phase and format select boxes and thinks out of line
![download](https://user-images.githubusercontent.com/6169940/28474994-943a598e-6e42-11e7-8344-0bcf98d8d523.PNG)

**Comment by stephaniesimms** [comment]
I imagine users will only want to download one phase at a time - but we can do usability testing for this page to confirm. I second @sjDCC suggestions for rephrasing:
- the 'questions as headings' concept is a difficult one to grasp. Could we rephrase this to "full question text and section headings"
- change 'supplimental section(s)...' to 'supplementary section(s)...'

**Comment by sjDCC** [comment]
Yeah, I suspect likewise that users will only want to download one phase at a time. Main use case for multiple phases at once is our auto-export for public DMPs

**Comment by JEK-III** [comment]
Wireframes updated. 

FWIW I don't see any problem with adding an "all phases" option to the dropdown menu if we decide at some point that it would be useful.

**Comment by sjDCC** [comment]
Perfect, thanks @JEK-III 

Like Steph, I don't expect there be much demand but having an all phases option in the dropdown is a sensible fix

**Comment by stephaniesimms** [comment]
@xsrust - wireframe is updated w/small text changes for Optional plan components. that should be the final tidbit for this issue.
![screen shot 2017-07-27 at 10 27 04 am](https://user-images.githubusercontent.com/12703638/28683681-515b7454-72b6-11e7-9ab8-b6b22570aa93.png)

**Comment by briri** [comment]
closing. All of the work on this one is complete
